gatewayconfig/bluetooth/characteristics/diagnostics_characteristic.py

```diff
@@ -77,7 +77,7 @@ def update_diagnostics_proto(self, p2pstatus):
                 logger.exception("Unexpected exception while trying to read p2pstatus")
                 raise ex
 
-        self.diagnostics_proto.diagnostics['ip'] = self.get_ip()
+        # self.diagnostics_proto.diagnostics['ip'] = self.get_ip()
```

Comment:
Did you mean to comment this?

Reply:
https://github.com/NebraLtd/hm-config/pull/136/files/f6407cc05f8c2b838568026495d242a9135a4f64..fc36a09cd27c9ae7f651bb107287833c9f6c086b#diff-8cb7c89f6dc236110ee7edb3865ce64db989a52833158107310a2e1c3a5fc663R118

https://github.com/NebraLtd/hm-config/pull/136/files/f6407cc05f8c2b838568026495d242a9135a4f64..fc36a09cd27c9ae7f651bb107287833c9f6c086b#diff-8cb7c89f6dc236110ee7edb3865ce64db989a52833158107310a2e1c3a5fc663R64